### DISCUSSION TITLE
Android Tablet for SDR ++

### DISCUSSION THREAD

**Discussion by Ralphthebrick** [AlexandreRouma/SDRPlusPlus#969] [General]
Hi, I am looking to upgrade an old tablet to something that will run SDR++.

Any recommendations on a model and spec that is up for the job ??

Thank you….Ralph

**Comment by sizzlik** [comment]
How old is old? Im using a Galaxy Tab A 2019 version with the RTL-SDR stick. I consider that one old, but it runs sdr++ perfectly fine and you can get it pretty cheap.
